perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -184,6 +184,53 @@ public RichMember call(ApiCaller ac, Deserializer params) throws PerunException
 		}
 	},
 
+	/*#
+	 * Sets the existing user as the sponsored user.
+	 *
+	 * Can be called with specific sponsor. If the parameter sponsor is null, then the user
+	 * which called this method will be set as a sponsor.
+	 *
+	 * @param vo int id of virtual organization
+	 * @param userToBeSponsored int id of user, that will be sponsored by sponsor
+	 * @param namespace String used for selecting external system in which guest user account will be created
+	 * @param password String password
+	 * @param sponsor int id of sponsoring user
+	 * @return RichMember sponsored member
+	 */
+	/*#
+	 * Sets the existing user as the sponsored user.
+	 *
+	 * Can be called with specific sponsor. If the parameter sponsor is null, then the user
+	 * which called this method will be set as a sponsor.
+	 *
+	 * @param vo int id of virtual organization
+	 * @param userToBeSponsored int id of user, that will be sponsored by sponsor
+	 * @param namespace String used for selecting external system in which guest user account will be created
+	 * @param password String password
+	 * @param sponsor int id of sponsoring user
```

Comment:
These two "javaDocs" look the same, I think you forgot to remove the sponsor from one of them.